planning/behavior_velocity_crosswalk_module/src/manager.cpp

```diff
@@ -141,25 +144,27 @@ void CrosswalkModuleManager::launchNewModules(const PathWithLaneId & path)
     // NOTE: module_id is always a lane id so that isModuleRegistered works correctly in the case
     //       where both regulatory element and non-regulatory element crosswalks exist.
     registerModule(std::make_shared<CrosswalkModule>(
-      node_, lane_id, reg_elem_id, lanelet_map_ptr, p, logger, clock_));
-    generateUUID(lane_id);
+      node_, road_lanelet_id, crosswalk_lanelet_id, reg_elem_id, lanelet_map_ptr, p, logger,
+      clock_));
+    generateUUID(crosswalk_lanelet_id);
     updateRTCStatus(
-      getUUID(lane_id), true, std::numeric_limits<double>::lowest(), path.header.stamp);
+      getUUID(crosswalk_lanelet_id), true, std::numeric_limits<double>::lowest(),
+      path.header.stamp);
   };
 
   const auto crosswalk_leg_elem_map = planning_utils::getRegElemMapOnPath<Crosswalk>(
     path, rh->getLaneletMapPtr(), planner_data_->current_odometry->pose);
 
   for (const auto & crosswalk : crosswalk_leg_elem_map) {
     // NOTE: The former id is a lane id, and the latter one is a regulatory element's id.
```

Comment:
@satoshi-ota 
question for my understanging
crosswalk.second.id(): id of the lanelet on the path 
crosswalk.first->crosswalkLanelet().id(): id of the crosswalk lanelet
crosswalk.first->id(): id of the crosswalk regulatory element

Reply:
Yes.